### DISCUSSION TITLE
GİTHUB SÜREKLİ PLEASE TRY AGAİN LATER SORUNU

### DISCUSSION THREAD

**Discussion by ersengoksan** [IBAX-io/go-ibax#1204] [Soru Cevap - Türkçe]
Merhaba, github görevlerinde star ve fork yapmaya çalışıyoruz tıklıyoruz dönüp ibax ana sayfaya geldiğimizde finished yaptığımız yukarıda belirttiğim please try again later sorunu ile karşılaşıyoruz çözüm bulan var mı?

**Comment by Yusufknk** [comment]
Yoğunluk söz konusu, lütfen aralarda tekrar deneyin.
